kubetest/e2e.go

```diff
@@ -413,8 +413,30 @@ func DumpFederationLogs(location string) error {
 }
 
 func PerfTest() error {
+	// TODO(wojtek-t): Remove once #2744 is debugged.
+	if out, err := output(exec.Command("ls ${GOPATH}/src")); err != nil {
```

Comment:
noted in the linked bug, but with context: the commands here should be separate args, not all one arg (this is looking for a binary named `ls ${GOPATH}/src`, rather than running `ls` with arg `${GOPATH/src`.

also, this should be `err == nil`.